scripts/superdesk-search/react.js

```diff
@@ -288,7 +288,7 @@
                      * @return {Boolean}
                      */
                     function hasThumbnail(item) {
-                        return item.type === 'picture' && item.renditions.thumbnail;
+                        return item.renditions.thumbnail;
```

Comment:
maybe it throws an error if `item.renditions` is undefined, so should be probably `item.renditions && item.renditions.thumbnail`
otherwise not sure why it would fail so much on travis